api/client/checkpoint.go

```diff
@@ -0,0 +1,52 @@
+// +build experimental
+
+package client
+
+import (
+	"fmt"
+
+	Cli "github.com/docker/docker/cli"
+	flag "github.com/docker/docker/pkg/mflag"
+	"github.com/docker/docker/runconfig"
+)
+
+// CmdCheckpoint checkpoints the process running in a container
+//
+// Usage: docker checkpoint CONTAINER
+func (cli *DockerCli) CmdCheckpoint(args ...string) error {
+	cmd := Cli.Subcmd("checkpoint", []string{"CONTAINER [CONTAINER...]"}, "Checkpoint one or more running containers", true)
```

Comment:
command description must go in https://github.com/docker/docker/blob/master/cli/common.go#L28
and used like this https://github.com/docker/docker/blob/master/api/client/attach.go#L18